code/day2-operations/deployment/k8s/otel-agent-mtls.yaml

```diff
@@ -140,19 +151,21 @@ spec:
       labels:
         app: opentelemetry
         component: otel-agent
+        sidecar.istio.io/inject: "true"        
```

Comment:
sidecar.istio.io/inject: "true"  needs now to be set as label instead of annotation